core/src/org/pentaho/platform/engine/core/audit/MessageTypes.java

```diff
@@ -78,4 +78,6 @@
 
   public static String ACTION_SEQUENCE_EXECUTE_CONDITIONAL = "condition"; //$NON-NLS-1$
 
+  public static String CANCELLED = "cancelled"; //$NON-NLS-1$
+
```

Comment:
ok, I will leave ALL other values 'as it is' since that was not my design and not my code and not my changes - for only "CANCELLED" status I will make it final.
Just to be sure not to break any legacy code that is actually relate on funny things like changing constants values, and my not be happy occasionously find that all message types just became final - just to have one more cancelled status.